db/db_impl_secondary.h

```diff
@@ -12,6 +12,17 @@
 #include "db/db_impl.h"
 
 namespace rocksdb {
+class LogReaderContainer {
```

Comment:
Can we have a constructor for this class?

Reply:
Done

Reply:
I should have been clearer. I mean a constructor in which we perform the creation of its members.